### DISCUSSION TITLE
LocalConnection

### DISCUSSION THREAD

**Discussion by robhenderson** [ruffle-rs/ruffle#2410] [Q&A]
Hi,

Does anyone know if implementation of LocalConnection is something that's likely to happen?

We've got an application that runs as several flash animations on the same page - and they rely on LocalConnection to communicate with each other.

Just wondering if it's worth holding out before thinking about rewriting the whole thing in HTML5!

Thanks.

**Comment by ROBERT-MCDOWELL** [comment]
If you can contribute things can accelerate. LocalConnection is also used with getRemote() method to get the sharedobject from the server which important indeed.

**Comment by ousia** [comment]
@robhenderson, you even posted a comment in the issue that tracks it (#259).

I’m not a developer, just an average computer user that reports bugs here from time to time.

If your issues are critical to you, your organization might [consider sponsoring @ruffle-rs](https://github.com/sponsors/ruffle-rs) for this implementation.

It is just my own take on the issue (or what I would consider doing if that were my case).

**Reply by robhenderson** [reply]
Sorry - didn't meant to offend anyone. I've only just discovered Ruffle so was just putting some feelers out. We'll probably look at redeveloping as nothing is critical - but will certainly keep in mind sponsorship of ruffle-rs too. Thanks.

**Reply by ousia** [reply]
Sorry, @robhenderson, your words were kind and perfectly fine.

Right now, there are a bunch of features that need to be implemented (only considering AS2).

I have my issues too (reported over months ago). But I realize that I will have to wait now, since other ones might come first.

Sponsorship came to my mind as a practical way to speed up the process.

**Reply by robhenderson** [reply]
Thanks @ousia, it's useful to know about the sponsorship option - so thanks for pointing it out.

Good luck with the issues you reported - hope you have some success soon.

**Comment by Bubby4j** [comment]
https://github.com/ruffle-rs/ruffle/pull/23212 would mostly implement this.
